stats/aggregated_stats.go

```diff
@@ -23,12 +23,9 @@ func convertToAggregatedStats(id string, containerIds map[string]string, resourc
 		aggregatedStats := []AggregatedStat{}
 		for _, stat := range stats {
 			if len(stat.Stats) > i {
-				if resourceType == "container" && id == "" {
-					id = stat.Name
-				}
```

Comment:
setting the id here resulted in all container stats getting the same containerId, as I was not resetting the id to "" after using the id.